seam/seam.py

```diff
@@ -4,7 +4,7 @@
 from typing_extensions import Self
 
 from seam.parse_options import parse_options
-from .routes import Routes
```

Comment:
I think every import inside `seam` should be a relative import, e.g.,

```
from .parse_options import parse_options
from .routes import Routes
```

If this is not working, you may need to add an `__init__.py` file in every sub directory. Try renaming `routes.py` to `__init.py__`

Reply:
a3f53443d511e314d1b6e8023a4ae61eb437765d